apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -3071,6 +3071,9 @@ int tc_filesystem_main(int argc, char *argv[])
 	tc_libc_stdio_tmpnam();
 	tc_libc_stdio_ungetc();
 	tc_libc_stdio_zeroinstream();
+#ifdef CONFIG_ITC_FS
+	itc_fs_main();
```

Comment:
There is no prototype. Doesn't it cause compilation warning (error)?

Reply:
No, sir.